lib/util/fs/index.ts

```diff
@@ -56,6 +56,13 @@ export async function readLocalSymlink(
   }
 }
 
+export async function writeFile(
+  fileName: string,
+  fileContent: string | Buffer,
+): Promise<void> {
+  await fs.outputFile(fileName, fileContent);
+}
+
```

Comment:
What's the reason for the addition if there already is `writeSystemFile` that does the exact same thing?

Reply:
Thanks for pointing this out, I've overlooked that such a function already exists. Also renamed `deleteFile`  to `deleteSystemFile` for consistency.